qa/automation/nikeev/Any Career Portal Page/Cookies.feature

```diff
@@ -0,0 +1,92 @@
+Feature: Cookies settings on any career portal page
+
+Scenario Outline: User sees cookies pop-up on any portal page
+	Given user previously didn't accept cookies for career portal
+	When user opens <name of the page> page
```

Comment:
I think the best way to resolve it is to use something like environment variables for all features. Because if we will change something in pages structure we must change every feature file (if we have links in Background) or every Scenario (if we have links in Examples)

Reply:
Ok, I agree